addons/arrays/fnc_findMin.sqf

```diff
@@ -24,13 +24,12 @@ Author:
 #include "script_component.hpp"
 SCRIPT(findMin);
 
-private ["_index"];
-
 if (!IS_ARRAY(_this)) exitWith {nil};
 if (_this isEqualTo []) exitWith {nil};
 
 params ["_min"];
-_index = 0;
+
+private _index = 0;
 
 {
     if (isNil "_x" || {(typeName _x) != (typeName 0)}) exitWith {_max = nil; _index = nil;};
```

Comment:
The function doesn't work at all, because a few lines below it does:
`if (isNil "_max") exitWith {nil};`
with `_max` never being defined (or carried over from another function due to missing `private`)

fixed in: #207